src/components/clusters/wizards/rosa/CreateRosaWizardFooter.jsx

```diff
@@ -60,11 +60,12 @@ const CreateRosaWizardFooter = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [steps, setStep]);
 
+  const oCMRoleLoading = values[FieldId.IsGetOCMRolePending] || false;
```

Comment:
I love what you are trying to do here, but there is no need to add any new FieldId variables.  All you need to do is call the react query for the ocmRole:

const { isPending: isGetOCMRolePending } = useFetchGetOCMRole(values[FieldId.AssociatedAwsId]);

Change `awsRequests.oCMRoleLoading` to `isGetOCMRolePending` and you should be good to go.
Note: I did not test this, but it should be close (you'll need to import the useFetchGetOCMRole hook)